Source/SpatialGDK/Private/SpatialActorChannel.cpp

```diff
@@ -241,6 +243,19 @@ bool USpatialActorChannel::ReplicateActor()
 
 	FMemMark MemMark(FMemStack::Get());	// The calls to ReplicateProperties will allocate memory on FMemStack::Get(), and use it in ::PostSendBunch. we free it below
 
+	USpatialTypeBinding* Binding = Interop->GetTypeBindingByClass(Actor->GetClass());
+	if (Binding)
+	{
+		bool bOldIsOwned = bIsNetOwned;
+		bIsNetOwned = Actor->GetNetConnection() != nullptr;
+
+		if (bIsNetOwned != bOldIsOwned || bFirstReplication)
+		{
+			Binding->UpdateEntityACL(this, bIsNetOwned);
```

Comment:
Is there a chance `UpdateEntityACL` will fail? There are some early out conditions there. If any of them prevent the update of ACL then it's wrong to flip `bFirstReplication`.